src/components/error/ErrorPage.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom";
+
+const ErrorPage = () => {
+  return (
+    <>
+      <h2>Whoopsie daisy</h2>
```

Comment:
The error message is very cute. Maybe also add something descriptive so the user knows what went wrong, eg. 'The page you are looking for does not exist' or 'This page cannot be found' 👍

Reply:
This is just a nitpick